src/cc_mm.c

```diff
@@ -82,6 +82,22 @@ _cc_realloc(void *ptr, size_t size, const char *name, int line)
     return p;
 }
 
+void *
+_cc_realloc_test(void *ptr, size_t size, const char *name, int line)
```

Comment:
Is it better to put the macro & alternative implementation inside _cc_realloc? There seems to be some duplicate in terms of logging, etc.